### PR TITLE
APEXMALHAR-2200 Enabled checkstyle for demos. Fixed checkstyle errors.

### PR DESCRIPTION
**Pull request by ShunxinLu**

### CODE REVIEW

File: demos/frauddetect/src/main/java/com/datatorrent/demos/frauddetect/SlidingWindowSumKeyVal.java [hunk 1]
```diff
@@ -69,31 +70,31 @@ public void processDataTuple(KeyValPair<K, V> tuple)
       }
       buffer.put(key, stateList);
     }
-
+
```
Comment (tweise) [inline demos/frauddetect/src/main/java/com/datatorrent/demos/frauddetect/SlidingWindowSumKeyVal.java]: Please remove trailing whitespaces. Should be an IDE setting.

File: demos/pom.xml [hunk 2]
```diff
@@ -163,9 +163,6 @@
         <plugin>
```
Comment (tweise) [inline demos/pom.xml]: Remove entire plugin configuration. It is inherited from parent pom.

Comment (siyuanh) [reply demos/pom.xml]: In parent, skip is set to true, shall we enforce the checkstyle check by default?

Comment (tweise) [reply demos/pom.xml]: Where do you see skip set to true? Yes, it should be on by default and problematic modules blacklisted.

File: demos/frauddetect/src/main/java/com/datatorrent/demos/frauddetect/MerchantTransactionBucketOperator.java [hunk 3]
```diff
@@ -102,34 +105,34 @@ private void processTuple(MerchantTransaction tuple)
...
   private void emitCreditCardKeyTuple(MerchantTransaction tuple, DefaultOutputPort<KeyValPair<MerchantKey, CreditCardData>> outputPort)
   {
     MerchantKey key = getMerchantKey(tuple);
-
+    
     CreditCardData data = new CreditCardData();
     data.fullCcNum = tuple.fullCcNum;
     data.amount = tuple.amount;
-
+
```
Comment (tweise) [inline demos/frauddetect/src/main/java/com/datatorrent/demos/frauddetect/MerchantTransactionBucketOperator.java]: trailing whitespace in many places

**Comment by tweise** [comment]
Great to see this cleanup happening! Please split the change into 2 commits. One for the build change and the second for the formatting changes. The second commit should be attributed to the CI user so there is no confusion when questions arise WRT the logic itself. You can do that by running:

```
git commit --amend --author "MalharJenkins <jenkins@datatorrent.com>"
```

**Pull request merged**